testing.py

```diff
@@ -1083,7 +1084,10 @@ class ExternalSearchTest(DatabaseTest):
     to ensure that it works well overall, with a realistic index.
     """
 
+    SIMPLIFIED_TEST_ELASTICSEARCH = os.environ.get('SIMPLIFIED_TEST_ELASTICSEARCH', u'http://localhost:9200')
```

Comment:
Do we need to set value of `SIMPLIFIED_TEST_ELASTICSEARCH` in .travis.yml?

Reply:
I think we should be explicit, but 9200 is the default Elasticsearch port so I'm not surprised this works without it.

Reply:
Yes, I agree. I think we should be as clear as possible whenever we reasonably can.

Reply:
Good call @vbessonov, I updated the travis file.